src/components/VolumeProperties.vue

```diff
@@ -54,28 +53,26 @@ export default defineComponent({
       () => !!cvrParams.value?.useVolumetricScatteringBlending
     );
 
-    const lightingModel = ref(2); // LAO is default
+    const lightingModel = ref(1);
```

Comment:
Leaving `lightingModel` as an integer is confusing. Let's change it to the corresponding key from `LIGHTING_MODELS`.

```
const lightingModel = ref<keyof typeof LIGHTING_MODELS>('hybrid');
```

The other comment later in this file indicates the changes necessary to support this.